apis/kueue/v1alpha1/tas_types.go

```diff
@@ -59,6 +59,10 @@ const (
 	// from the Job's PodTemplate also have the label. For the Pod-based
 	// integrations the label is added in webhook during the Pod creation.
 	TASLabel = "kueue.x-k8s.io/tas"
+
+	// PodGroupPodIndexLabel is a label set on the Pod's metadata belonging
+	// to a Pod group. It indicates the Pod's index within the group.
+	PodGroupPodIndexLabel = "kueue.x-k8s.io/pod-group-pod-index"
```

Comment:
nit: please also update the KEP with the new label

Reply:
Done